commands/searchExploit.go

```diff
@@ -120,48 +131,47 @@ func (p *SearchExploitCmd) Execute(_ context.Context, f *flag.FlagSet, _ ...inte
 		}
 		return subcommands.ExitFailure
 	}
-	
+
 	stype := c.SearchConf.SearchType
 	param := c.SearchConf.SearchParam
 
-	var result = []models.Exploit{}
-	if stype == "All" {
+	var results = []*models.Exploit{}
+	switch stype {
+	case "All":
 		if param != "" {
 			log15.Error("A SearchType [All] does not require parameters")
 			return subcommands.ExitFailure
 		}
-		result = driver.GetExploitAll()
-	} else if stype == "CVE" {
-		r := regexp.MustCompile(`^(\d{1,}-\d{1,})`)
-		if !r.Match([]byte(param)) {
+		results = driver.GetExploitAll()
+	case "CVEID":
```

Comment:
Not CVE ID, CVE is correct seems to be better.
search type are All or CVE or ID.